tensorflow_serving/batching/batching_session.cc

```diff
@@ -102,6 +105,127 @@ TensorSignature TensorSignatureFromSignatureDefs(
   return tensor_signature;
 }
 
+template<int num_dims>
```

Comment:
Please be sure every function, class & method has a comment explaining briefly what it does.